### PR TITLE
feat(ui): add in-product notifications

### PR DESCRIPTION
**Pull request by RichardKruemmel**

### CODE REVIEW

File: src/features/notifications/checkNotifications.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "lastSeenNotificationId",
+    0,
+  );
+  useEffect(() => {
+    if (!authenticated) {
+      return;
+    }
+    // We delay the notification to ensure that the Toaster component (in layout.tsx L491) is mounted before we call the toast function
+    const timeoutId = setTimeout(() => {
+      notification.reverse().forEach((n) => {
```
Comment (maxdeichmann) [inline src/features/notifications/checkNotifications.tsx:50]: can you remove this?

File: src/features/tag/components/TagMananger.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ const TagManager = ({
             setSelectedTags={setSelectedTags}
           />
           <CommandList
-            className={availableTags.length > 0 ? "mt-2" : undefined}
+            className={`overflow-auto ${availableTags.length > 0 ? "mt-2" : ""}`}
```
Comment (maxdeichmann) [inline src/features/tag/components/TagMananger.tsx:64]: This is the change for the long tags, correct?

Comment (RichardKruemmel) [reply src/features/tag/components/TagMananger.tsx:64]: Yes 🙂

File: src/features/notifications/checkNotifications.tsx (hunk 1 above)
Comment (RichardKruemmel) [reply src/features/notifications/checkNotifications.tsx:50]: You mean the comment?

File: src/features/notifications/CloseIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const CloseIcon = () => (
+  <svg width="14" height="14" viewBox="0 0 16 16" fill="currentColor">
```
Comment (maxdeichmann) [inline src/features/notifications/CloseIcon.tsx:2]: wo hattest du das hier her? Wir nutzen normalerweise lucide für icons, kannst du das hauch hier verwenden?

Comment (RichardKruemmel) [reply src/features/notifications/CloseIcon.tsx:2]: Ich hatte das von dem Sonner repo genommen um die UI zu kopieren.

**Review by marcklingen — CHANGES_REQUESTED** [review]
I get this error when running `npm run dev`

```
Error reading from local storage ReferenceError: localStorage is not defined
    at eval (webpack-internal:///./src/components/useLocalStorage.tsx:28:33)
    at useReducer (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5079:57)
    at Object.useState (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5017:10)
    at useState (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react/cjs/react.development.js:1622:21)
    at useLocalStorage (webpack-internal:///./src/components/useLocalStorage.tsx:26:78)
    at useCheckNotification (webpack-internal:///./src/features/notifications/checkNotifications.tsx:121:116)
    at Layout (webpack-internal:///./src/components/layouts/layout.tsx:87:106)
    at renderWithHooks (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5658:16)
    at renderIndeterminateComponent (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5731:15)
    at renderElement (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:5946:7)
    at renderNodeDestructiveImpl (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6104:11)
    at renderNodeDestructive (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6076:14)
    at renderNode (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6259:12)
    at renderChildrenArray (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6211:7)
    at renderNodeDestructiveImpl (/Users/xxx/repos/github/langfuse/langfuse/node_modules/react-dom/cjs/react-dom-server.browser.development.js:6141:7)
```

**Comment by RichardKruemmel** [comment]
Did you run npm i prior?

**Comment by marcklingen** [comment]
yes, same error gets also logged during build. I think localStorage is only available client side, thus we need to check if window is available in the useEffect. I think you can find a default implementation for nextjs for this.

**Comment by RichardKruemmel** [comment]
Interesting, I will look into it! Thanks for reporting 🙏🏻

**Review by marcklingen — APPROVED** [review]

**Pull request merged**